### DISCUSSION TITLE
show private contributions

### DISCUSSION THREAD

**Discussion by AidinHamedi** [anuraghazra/github-readme-stats#3321] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

No.

### Describe the solution you'd like

I want the card to show the privet contributions too.

**Comment by qwerty541** [comment, marked as answer]
Hey, @Aydinhamedi! By default, the stats card only shows statistics only from public repositories. To show private statistics on the stats card, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token. Our documentation contains that information.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d2745ae3-8a5b-47f5-b776-9d0594c2e60f)

**Comment by AidinHamedi** [comment]
Ok thanks

**Comment by dingrobu** [comment]
Hi all

I managed to deploy my own instance on vercel. However I am not seeing my private contributions using this code chunk (of course with my user name and my own app link):

`[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)`

What could be the issue with this?

`The domains generated by vercel link to the repository https://github.com/anuraghazra/github-readme-stats. `

Could this be the issue?

Thanks in advance

**Reply by qwerty541** [reply]
Answered in https://github.com/anuraghazra/github-readme-stats/discussions/3366#discussioncomment-7285896, check it out.
